observation_portal/proposals/admin.py

```diff
@@ -91,10 +91,12 @@ def proposal_title(self, obj):
 
 class ProposalInviteAdmin(admin.ModelAdmin):
     model = ProposalInvite
+    raw_id_fields = ['proposal']
```

Comment:
do raw_id_fields still give you a link to that admin object?

Reply:
Yep!